mllib-local/src/test/scala/org/apache/spark/ml/stat/distribution/MultivariateGaussianSuite.scala

```diff
@@ -31,11 +31,15 @@ class MultivariateGaussianSuite extends SparkMLFunSuite {
     val mu = Vectors.dense(0.0)
     val sigma1 = Matrices.dense(1, 1, Array(1.0))
     val dist1 = new MultivariateGaussian(mu, sigma1)
+    assert(dist1.logpdf(x1) ~== -0.9189385332046727 absTol 1E-5)
```

Comment:
numbers here are generated in repl in 2.4.5.

![image](https://user-images.githubusercontent.com/7322292/75755322-5f4c3300-5d69-11ea-85d4-807c6d50f6f4.png)